vignettes/distributions.Rmd

```diff
@@ -26,13 +26,19 @@ The currently available parametric delay distributions are continuous probabilit
 
-To discretise the continuous probability distribution into discrete probabilities $p^{\prime}_{g,t,d}$, we use its cumulative distribution function $F^{\mu_{g,t}, \upsilon_{g,t}}$ to compute
-$$p^{\prime}_{g,t,d} = \frac{F^{\mu_{g,t}, \upsilon_{g,t}}(d+1) - F^{\mu_{g,t}, \upsilon_{g,t}}(d)}{F^{\mu_{g,t}, \upsilon_{g,t}}(D)}.$$
-In words, the discrete probability mass function (pmf) is obtained from the continuous distribution by dividing it into $D$ different bins $$(0,1],\, (1,2],\, \ldots,\, (D-1,D],$$
-where $D$ is the maximum delay covered by the model. Importantly, the bins are normalized by $F^{\mu_{g,t}, \upsilon_{g,t}}(D)$, which ensures that the $p^{\prime}_{g,t,d}$ sum to 1. Since $F^{\mu_{g,t}, \upsilon_{g,t}}(D)$ is the probability of reporting before the maximum delay, this can also be interpreted as conditioning our distribution on the maximum delay.
+The exact form of this discretisation is complex due to the interaction between primary and secondary events. Rather than modelling this explicitly we approximate it by assuming a uniform censoring interval for each delay of 2 days. This comes from assuming daily censoring for both the primary and secondary event that make up the delay distribution and ignoring the interaction between these censoring intervals. This means that the probability of reporting a delay of $d$ days is the probability of reporting a delay of $d+1$ days or less, minus the probability of reporting a delay of $d-1$ days or less. This is then normalised by the sum of the probabilities of reporting a delay of $d$ for all delays up to some maximum delay, $D$. 
 
-Note that because of the discretisation and normalization, the discrete random variable we obtain generally does not have exactly the same moments as the original, continuous distribution.
+More rigorously, we can define this in terms of the cumulative distribution function. Let $F^{\mu_{g,t}, \upsilon_{g,t}}$ be the cumulative distribution function of the continuous probability distribution with parameters $\mu_{g,t}$ and $\upsilon_{g,t}$. Then the probability of reporting a delay of $d$ days is
+$$p_{g,t,d} = \frac{F^{\mu_{g,t}, \upsilon_{g,t}}(d+1) - F^{\mu_{g,t}, \upsilon_{g,t}}(d-1)}{F^{\mu_{g,t}, \upsilon_{g,t}}(D + 1 ) + F^{\mu_{g,t}, \upsilon_{g,t}}(D)}.$$
+
+Unless $d = 0$ then we instead have
+
+$$p_{g,t,1} = \frac{F^{\mu_{g,t}, \upsilon_{g,t}}(1)}{F^{\mu_{g,t}, \upsilon_{g,t}}(D + 1 ) + F^{\mu_{g,t}, \upsilon_{g,t}}(D)}.$$
+
+Normalising by $F^{\mu_{g,t}, \upsilon_{g,t}}(D+1) + F^{\mu_{g,t}, \upsilon_{g,t}}(D)$, ensures that the $p^{\prime}_{g,t,d}$ sum to 1. Since $F^{\mu_{g,t}, \upsilon_{g,t}}(D)$ is the probability of reporting before the maximum delay, this can also be interpreted as conditioning our distribution on the maximum delay.
```

Comment:
yes agree. If D was infinite we would need to divide by 2 (which is true as well I guess with the 1 day step but you obviously don't need to divide everything by one).

Reply:
It would be awesome if we could at least conceptually separate these two parts from each other when explaining this in the documentation. Like, deriving the normalization factor for an infinite D first, and then adding the conditioning on D.

Reply:
Do you think we can make this its own issue/PR pair to resolve/improve?